common/test/acceptance/pages/studio/settings_graders.py

```diff
@@ -108,6 +117,89 @@ def change_assignment_name(self, old_name, new_name):
         self.q(css='#course-grading-assignment-name').filter(
             lambda el: el.get_attribute('value') == old_name).fill(new_name)
 
+    def set_weight(self, assignment_name, weight):
+        """
+        Set the weight of the assignment type.
+        Arguments:
+            assignment_name(string): Assignment name for which weight is to be changed.
+            weight(string): New weight
+        """
+        weight_id = '#course-grading-assignment-gradeweight'
+        # index = self._get_type_index(assignment_name)
+        f = self.q(css=weight_id).results[-1]
+        # assert index != -1
+        for __ in xrange(len(assignment_name)):
+            f.send_keys(Keys.END, Keys.BACK_SPACE)
+        f.send_keys(weight)
+
+    def get_assignment_weight(self, assignment_name):
+        """
+        Gets the weight of assignment
+        Arguments:
+            assignment_name (str): Name of the assignment
+        Returns:
+            string: Weight of the assignment
+        """
+        self.wait_for_element_visibility(
+            '#course-grading-assignment-gradeweight',
+            'Weight fields are present'
+        )
+        weight_id = '#course-grading-assignment-gradeweight'
+        index = self._get_type_index(assignment_name)
+        all_weight_elements = self.q(css=weight_id).results
+        return all_weight_elements[index].get_attribute('value')
+
+    def is_notification_button_disbaled(self):
+        """
+        Check to see if notification button is disabled.
+        Returns: True if button is disabled.
```

Comment:
Please follow the edX coding guidelines for docstrings: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings